package.json

```diff
@@ -1,7 +1,11 @@
 {
   "name": "@onebeyond/agnostic-cms-harmonizer",
-  "version": "1.0.0",
+  "version": "0.1.0",
```

Comment:
can we set back to `v1` please?

Reply:
It will be v1 when the action is triggered. I changed it to this version so it's bumped to 1.0.0